pyproject.toml

```diff
@@ -96,6 +98,8 @@ ignore = [
     "E203",
     # line-too-long (due to Copyright header)
     "E501",
+    # [*] Import block is un-sorted or un-formatted
+    "I001",
```

Comment:
Why is this rule ignored? Isn't using this the whole point of turning on the sorting functionality?

Reply:
wanted to keep the diff minimal and just show the parity, so I may suggest landing this with this exception and then remove this line and apply related changes... or do you prefer to have it at once?

Reply:
I'm confused, what do you mean by parity? With this check disabled, won't we have strictly less linting done now than before? Unless I'm severely misunderstanding, disabling I001 disables the primary functionality of isort when running ruff: sorting imports in blocks.

Reply:
so for now `isort` does not flag any issue so I set the Ruff so it does not either even it seem that Ruff found some issue...
yes, I agree, so I enable that rule and see how we like the formatting result and eventually close this PR if the formating is not liked :)

Reply:
My assumption was that without the check enabled ruff simply didn't find any issues, so no changes were made. It wouldn't be good if we merged the change and preserved the current sorting, only to find that nothing was getting sorted going forward :slightly_smiling_face: